### ISSUE TITLE
linter-gcc erroneously reports `Using built-in specs. in <file>` as an error for each file

### ISSUE THREAD

**Issue by meiower** [hebaishi/linter-gcc#87]
## Checklist
* [ Y ] Check that you have the GNU C Compiler installed by typing ```gcc --version``` into a terminal.
* [ - ] If you don't have the ```linter``` package installed, either update to the latest version of ```linter-gcc``` or install linter directly with ```apm install linter```.
* [ Y ] Ensure that you are using a valid grammar - either C/C++/C++14
* [ Y ] Ensure that you have read all the relevant information in the [wiki](https://github.com/hebaishi/linter-gcc/wiki), particularly the [Common Issues](https://github.com/hebaishi/linter-gcc/wiki/Common-Issues) page.

## Versions

- **gcc-linter**:  0.6.15
- **linter**:  1.11.18
- **Atom**:  1.8.0
- _No updates outstanding_

## Issue (including debugging messages):

linter-gcc reports `Using built-in specs. in <file>` as an error for each file; however that output is not prefixed with the GCC error prepend string `error` - it's purely informational, not one of `error`, `warning` or `note`.

Note: this is an embedded project, under Platform.io.

**Output from Developer Console after enabling Debugging Messages:**
```
linter-gcc: Reading settings from: /home/bob/TC Home/Projects/Hardware and Embedded/201505 Bike Rave - Spoke Lights/SpokeLight_Sim_CppAllegro/.gcc-flags.json
/home/bob/.atom/packages/linter-gcc/lib/utility.js:113 linter-gcc config: {"execPath":"/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/arm-none-eabi-g++","gccIncludePaths":"/home/bob/.platformio/packages/framework-arduinosam/cores/adafruit_feather_m0,/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/CMSIS/Include,/home/bob/.platformio/packages/framework-arduinosam/system/libsam,/home/bob/.platformio/packages/framework-arduinosam/system/libsam/include,/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL,/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL/d21g18a/include,/home/bob/.platformio/packages/framework-arduinosam/variants/adafruit_feather_m0,/home/bob/TC Home/Projects/Hardware and Embedded/201505 Bike Rave - Spoke Lights/SpokeLight_Sim_CppAllegro,/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/HID,/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/SAMD_AnalogCorrection/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/SPI,/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/USBHost/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/Wire,/home/bob/.platformio/packages/framework-arduinosam/libraries/Audio/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Bridge/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Esplora/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Ethernet/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Firmata/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/GSM/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/HID/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/LiquidCrystal/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/RobotIRremote/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Robot_Control/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Robot_Motor/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/SD/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/SPI/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Scheduler/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Servo/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/SpacebrewYun/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Stepper/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/TFT/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Temboo/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/USBHost/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/WiFi/src,/home/bob/.platformio/packages/framework-arduinosam/libraries/Wire/src,/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/arm-none-eabi/include,/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/lib/gcc/arm-none-eabi/4.8.4/include-fixed,/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/lib/gcc/arm-none-eabi/4.8.4/include","gccSuppressWarnings":false,"gccDefaultCFlags":"-fsyntax-only -std=gnu11 -g -Os -ffunction-sections -fdata-sections -Wall -mthumb -mcpu=cortex-m0plus -nostdlib --param max-inline-insns-single=500 -v -DF_CPU=48000000L -DUSBCON -DPLATFORMIO=030100 -DARDUINO_SAMD_FEATHER_M0 -DARDUINO_ARCH_SAMD -D__SAMD21G18A__ -DARDUINO=10608 -DUSB_VID=0x239A -DUSB_PID=0x800B -DUSB_PRODUCT=\"Adafruit\\ Feather\\ M0\" -DUSB_MANUFACTURER=\"Adafruit\"","gccDefaultCppFlags":"-fsyntax-only -fno-rtti -fno-exceptions -fno-threadsafe-statics -std=c++11 -g -Os -ffunction-sections -fdata-sections -Wall -mthumb -mcpu=cortex-m0plus -nostdlib --param max-inline-insns-single=500 -v -DF_CPU=48000000L -DUSBCON -DPLATFORMIO=030100 -DARDUINO_SAMD_FEATHER_M0 -DARDUINO_ARCH_SAMD -D__SAMD21G18A__ -DARDUINO=10608 -DUSB_VID=0x239A -DUSB_PID=0x800B -DUSB_PRODUCT=\"Adafruit\\ Feather\\ M0\" -DUSB_MANUFACTURER=\"Adafruit\"","gccErrorLimit":15}
/home/bob/.atom/packages/linter-gcc/lib/utility.js:165 linter-gcc: /home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/arm-none-eabi-g++ -fsyntax-only -fno-rtti -fno-exceptions -fno-threadsafe-statics -std=c++11 -g -Os -ffunction-sections -fdata-sections -Wall -mthumb -mcpu=cortex-m0plus -nostdlib --param max-inline-insns-single=500 -v -DF_CPU=48000000L -DUSBCON -DPLATFORMIO=030100 -DARDUINO_SAMD_FEATHER_M0 -DARDUINO_ARCH_SAMD -D__SAMD21G18A__ -DARDUINO=10608 -DUSB_VID=0x239A -DUSB_PID=0x800B -DUSB_PRODUCT="Adafruit Feather M0" -DUSB_MANUFACTURER="Adafruit" -fmax-errors=15 -I/home/bob/.platformio/packages/framework-arduinosam/cores/adafruit_feather_m0 -I/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/CMSIS/Include -I/home/bob/.platformio/packages/framework-arduinosam/system/libsam -I/home/bob/.platformio/packages/framework-arduinosam/system/libsam/include -I/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL -I/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL/d21g18a/include -I/home/bob/.platformio/packages/framework-arduinosam/variants/adafruit_feather_m0 -I/home/bob/TC Home/Projects/Hardware and Embedded/201505 Bike Rave - Spoke Lights/SpokeLight_Sim_CppAllegro -I/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/HID -I/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/SAMD_AnalogCorrection/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/SPI -I/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/USBHost/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/__cores__/adafruit_feather_m0/Wire -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Audio/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Bridge/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Esplora/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Ethernet/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Firmata/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/GSM/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/HID/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/LiquidCrystal/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/RobotIRremote/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Robot_Control/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Robot_Motor/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/SD/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/SPI/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Scheduler/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Servo/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/SpacebrewYun/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Stepper/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/TFT/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Temboo/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/USBHost/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/WiFi/src -I/home/bob/.platformio/packages/framework-arduinosam/libraries/Wire/src -I/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/arm-none-eabi/include -I/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/lib/gcc/arm-none-eabi/4.8.4/include-fixed -I/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/lib/gcc/arm-none-eabi/4.8.4/include /tmp/624170c6-b68c-4ed1-b469-4f7e742a2db2.cpp
```

**Output from build with -v option showing sample compile and GCC output:**
_(Seems lots of other non- error/warning/note lines are correctly ignored!)_
```
Compiling .pioenvs/adafruit_feather_m0_usb/src/main_platformio.o
Using built-in specs.
COLLECT_GCC=arm-none-eabi-g++
Target: arm-none-eabi
Configured with: /home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/src/gcc/configure --target=arm-none-eabi --prefix=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/install-native --libexecdir=/home/bui
ld/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/install-native/lib --infodir=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/install-native/share/doc/gcc-arm-none-eabi/info --mandir=/home/build/toolchain/gcc-a
rm-none-eabi-4_8-2014q3-20140805/install-native/share/doc/gcc-arm-none-eabi/man --htmldir=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/install-native/share/doc/gcc-arm-none-eabi/html --pdfdir=/home/build/toolc
hain/gcc-arm-none-eabi-4_8-2014q3-20140805/install-native/share/doc/gcc-arm-none-eabi/pdf --enable-languages=c,c++ --enable-plugins --disable-decimal-float --disable-libffi --disable-libgomp --disable-libmudflap --disable-li
bquadmath --disable-libssp --disable-libstdcxx-pch --disable-nls --disable-shared --disable-threads --disable-tls --with-gnu-as --with-gnu-ld --with-newlib --with-headers=yes --with-python-dir=share/gcc-arm-none-eabi --with-
sysroot=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/install-native/arm-none-eabi --build=x86_64-linux-gnu --host=x86_64-linux-gnu --with-gmp=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/build-native/host-libs/usr --with-mpfr=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/build-native/host-libs/usr --with-mpc=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/build-native/host-libs/usr --with-
isl=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/build-native/host-libs/usr --with-cloog=/home/build/toolchain/gcc-arm-none-eabi-4_8-2014q3-20140805/build-native/host-libs/usr --with-libelf=/home/build/toolcha
in/gcc-arm-none-eabi-4_8-2014q3-20140805/build-native/host-libs/usr --with-host-libstdcxx='-static-libgcc -Wl,-Bstatic,-lstdc++,-Bdynamic -lm' --with-pkgversion='GNU Tools for ARM Embedded Processors' --with-multilib-list=ar
mv6-m,armv7-m,armv7e-m,armv7-r
Thread model: single
gcc version 4.8.4 20140725 (release) [ARM/embedded-4_8-branch revision 213147] (GNU Tools for ARM Embedded Processors)
COLLECT_GCC_OPTIONS='-o' '.pioenvs/adafruit_feather_m0_usb/src/main_platformio.o' '-c' '-fno-rtti' '-fno-exceptions' '-fno-threadsafe-statics' '-std=c++11' '-g' '-Os' '-ffunction-sections' '-fdata-sections' '-Wall' '-mthumb'
 '-mcpu=cortex-m0plus' '-nostdlib' '--param' 'max-inline-insns-single=500' '-v' '-D' 'F_CPU=48000000L' '-D' 'USBCON' '-D' 'PLATFORMIO=030100' '-D' 'ARDUINO_SAMD_FEATHER_M0' '-D' 'ARDUINO_ARCH_SAMD' '-D' '__SAMD21G18A__' '-D'
 'ARDUINO=10608' '-D' 'USB_VID=0x239A' '-D' 'USB_PID=0x800B' '-D' 'USB_PRODUCT="Adafruit Feather M0"' '-D' 'USB_MANUFACTURER="Adafruit"' '-I' '/home/bob/.platformio/packages/framework-arduinosam/cores/adafruit_feather_m0' '-
I' '/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/CMSIS/Include' '-I' '/home/bob/.platformio/packages/framework-arduinosam/system/libsam' '-I' '/home/bob/.platformio/packages/framework-arduinosam/system/li
bsam/include' '-I' '/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL' '-I' '/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL/d21g18a/include' '-I' '/home/bob/.platfor
mio/packages/framework-arduinosam/variants/adafruit_feather_m0' '-I' '.'
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/cc1plus -quiet -v -I /home/bob/.platformio/packages/framework-arduinosam/cores/adafruit_feather_m0 -I /home/bob/.platformio/packages/
framework-arduinosam/system/CMSIS/CMSIS/Include -I /home/bob/.platformio/packages/framework-arduinosam/system/libsam -I /home/bob/.platformio/packages/framework-arduinosam/system/libsam/include -I /home/bob/.platformio/packa
ges/framework-arduinosam/system/CMSIS/Device/ATMEL -I /home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL/d21g18a/include -I /home/bob/.platformio/packages/framework-arduinosam/variants/adafruit_fea
ther_m0 -I . -imultilib armv6-m -iprefix /home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/ -isysroot /home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../arm-none-eabi -D__
USES_INITFINI__ -D F_CPU=48000000L -D USBCON -D PLATFORMIO=030100 -D ARDUINO_SAMD_FEATHER_M0 -D ARDUINO_ARCH_SAMD -D __SAMD21G18A__ -D ARDUINO=10608 -D USB_VID=0x239A -D USB_PID=0x800B -D USB_PRODUCT="Adafruit Feather M0" -D
 USB_MANUFACTURER="Adafruit" main_platformio.cpp -quiet -dumpbase main_platformio.cpp -mthumb -mcpu=cortex-m0plus -auxbase-strip .pioenvs/adafruit_feather_m0_usb/src/main_platformio.o -g -Os -Wall -std=c++11 -version -fno-rt
ti -fno-exceptions -fno-threadsafe-statics -ffunction-sections -fdata-sections --param max-inline-insns-single=500 -o /tmp/ccM1XVvb.s
GNU C++ (GNU Tools for ARM Embedded Processors) version 4.8.4 20140725 (release) [ARM/embedded-4_8-branch revision 213147] (arm-none-eabi)
compiled by GNU C version 4.3.2, GMP version 4.3.2, MPFR version 2.4.2, MPC version 0.8.1
GGC heuristics: --param ggc-min-expand=100 --param ggc-min-heapsize=131072
ignoring duplicate directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/../../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include/c++/4.8.4"
ignoring duplicate directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/../../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include/c++/4.8.4/arm-none-eabi/armv6-m"
ignoring duplicate directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/../../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include/c++/4.8.4/backward"
ignoring duplicate directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/../../lib/gcc/arm-none-eabi/4.8.4/include"
ignoring nonexistent directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../arm-none-eabi/usr/local/include"
ignoring duplicate directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/../../lib/gcc/arm-none-eabi/4.8.4/include-fixed"
ignoring duplicate directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/../../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include"
ignoring nonexistent directory "/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../arm-none-eabi/usr/include"
ignoring nonexistent directory "/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL/d21g18a/include"
#include "..." search starts here:
#include <...> search starts here:
/home/bob/.platformio/packages/framework-arduinosam/cores/adafruit_feather_m0
/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/CMSIS/Include
/home/bob/.platformio/packages/framework-arduinosam/system/libsam
/home/bob/.platformio/packages/framework-arduinosam/system/libsam/include
/home/bob/.platformio/packages/framework-arduinosam/system/CMSIS/Device/ATMEL
/home/bob/.platformio/packages/framework-arduinosam/variants/adafruit_feather_m0
.
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include/c++/4.8.4
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include/c++/4.8.4/arm-none-eabi/armv6-m
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include/c++/4.8.4/backward
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/include
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/include-fixed
/home/bob/.platformio/packages/toolchain-gccarmnoneeabi/bin/../lib/gcc/arm-none-eabi/4.8.4/../../../../arm-none-eabi/include
End of search list.
GNU C++ (GNU Tools for ARM Embedded Processors) version 4.8.4 20140725 (release) [ARM/embedded-4_8-branch revision 213147] (arm-none-eabi)
compiled by GNU C version 4.3.2, GMP version 4.3.2, MPFR version 2.4.2, MPC version 0.8.1
GGC heuristics: --param ggc-min-expand=100 --param ggc-min-heapsize=131072
Compiler executable checksum: 4b27e8b89ba0e00502d5e8ae7be4d430

```
